specs/Standards/NonFungibleToken/payouts.md

```diff
@@ -0,0 +1,58 @@
+# Standard for a Multiple-Recipient-Payout mechanic on NFT Contracts
+
+A standard for supporting structures that allow for NFT's to inform financial
+contracts to payout multiple addresses on sales.
```

Comment:
```suggestion
contracts to pay out multiple addresses on sales.
```

"Payout" is a noun; "pay out" is the verb. (Similar to "What's your login?" vs "Please log in")